src/main/resources/schema/ans/0.10.3/utils/focal_point.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.3/utils/focal_point.json",
+  "description": "Coordinates representing the 'visual center' of an image. The X axis is horizontal line and the Y axis the vertical line, with 0,0 being the LEFT, TOP of the image.",
+  "type": "object",
+  "properties": {
```

Comment:
I would just move these fields into the trait_focal_point file, I don't think there's a need for a separate file